.github/workflows/workflow.yml

```diff
@@ -45,13 +48,52 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
```

Comment:
Just to clean up. I think it was put in there because of some depend packaging issue in the past that no longer is a problem. Generally not in any of the example/documentation workflows.

In this workflow, packages should already be rolling with the underlying deployments, just not as aggressively as forcing an upgrade.

Unless you really prefer it, I'm inclined to remove it.

Reply:
Ok got it. I just keep seeing lines like this elsewhere to make sure that pip is up to date (to solve issues with newer packages requiring certain versions of pip, such as for manylinux wheels, etc.). But it shouldn't be an issue here.